src/components/AssetCheckoutScanner.tsx

```
@@ -0,0 +1,80 @@
+          setScannedAsset(decodedText);
+          toast.success(`Asset Scanned: ${decodedText}`);
+          scanner.clear(); // Briefly stop scanning to let the UI update
+        } else if (!scannedUser) {
+          setScannedUser(decodedText);
+          toast.success(`User Scanned: ${decodedText}`);
+          scanner.clear();
+        }
+      },
+      (error) => {
+        // We ignore scan errors because it throws them constantly until it finds a valid code
+      }
```

ESLint

[error] Insert `,`

(prettier/prettier)